host/CMakeLists.txt

```diff
@@ -337,8 +392,24 @@ if (UNIX)
       -Wl,-z,noexecstack)
 endif ()
 
+add_library(oehostverifyapp INTERFACE)
```

Comment:
Do we need two versions of the `oehostverify` library given the use case? Should the dynamic export version just be the default version if that's needed? I don't believe we publish `oehostapp` in the SDK today, and it's consumed by tests only.

Reply:
The dynamic library could be the default since I don't see the point of the static one. The `oehostapp` is part of the export list, so I think it's a usable target by the external consumers.

Reply:
Made the dynamic export the default. That's actually how it is in our pkgconfig too.